run_test.sh

```diff
@@ -10,7 +10,7 @@ if [[ ! -f $VENV ]]; then
     then
         echo "Installing dbt-spark"
         pip install dbt-spark[ODBC] --upgrade --pre
-        pip install pyodbc==4.0.32  # See https://github.com/dbt-labs/dbt-external-tables/issues/156
+        # pip install pyodbc==4.0.32  # See https://github.com/dbt-labs/dbt-external-tables/issues/156
```

Comment:
Since it doesn't appear to be required anymore, I'm going to peel this out. We can add it back if needed down the road.
```suggestion
```